src/tasks/minions/miningActivity.ts

```diff
@@ -93,7 +92,23 @@ export default class extends Task {
 			}
 		}
 
-		str += `\n\nYou received: ${await createReadableItemListFromBank(this.client, loot)}.`;
+		if (ore.name.toLowerCase() === 'gem rock') {
+			const gemLoot = new JSLoot();
+			for (let i = 0; i < quantity; i++) {
+				gemLoot.add(ore.gems!.roll());
+			}
+			await user.addItemsToBank(gemLoot.values(), true);
+			str += `\n\nYou received: ${await createReadableItemListFromBank(
+				this.client,
+				gemLoot.values()
```

Comment:
just add this to the current `loot` var, let everything thats already in place take care of the rest

Reply:
you literally did the exact opposite of what i said and then marked this as resolved???